Documentation/operations/upgrade.rst

```diff
@@ -965,7 +1262,7 @@ New ConfigMap Options
     available. Cilium now uses CRDs by default which limits the use of
     well-known identities to the managed etcd mode. With the addition of this
     option, well-known identities are disabled by default in all new deployment
-    and only enabled if the Helm option ``etcd.managed=true`` is set. Consider
+    and only enabled if the Helm option ``global.etcd.managed=true`` is set. Consider
```

Comment:
was this supposed to be modified?

Reply:
this is  referring to a previous version where global should be included